File: surfaceview/src/main/java/com/by5388/xw/android/surfaceview/MainActivity.java
package com.by5388.xw.android.surfaceview;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.Window;
import android.view.WindowManager;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Window window = getWindow();
        //屏幕常亮
        window.addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON);
        setContentView(R.layout.activity_main);

    }
}

File: kotlinDemo/src/main/java/com/by5388/xw/android/kotlindemo/MainActivity.kt
package com.by5388.xw.android.kotlindemo

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.support.v7.widget.LinearLayoutManager
import com.by5388.xw.android.kotlindemo.adapter.PersonAdapter
import com.by5388.xw.android.kotlindemo.bean.Person
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        initData()
    }

    private fun initData() {
        var people: MutableList<Person> = ArrayList()
        people.add(Person("a", 20))
        people.add(Person("b", 30))
        var adapter = PersonAdapter(this, people)
        recycler_view.layoutManager = LinearLayoutManager(this)
        recycler_view.adapter = adapter
    }
}
